docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,407 @@
+    - Quickpanel has been added to WindowSystem.
+  - Tizen.Network.Connection
+    - Required privilege level has been changed.
+  - Tizen.MachineLearning
+    - Custom filter class has been added.
+  - Tizen.Multimedia
+    - Setter/Getter for volume recording has been added to AudioIO.
+    - Result code and playback state have been added to MediaController.
+    - New RequestCommand for async has been added to MediaController.
+    - New barcode types have been added to Vision.
+  - Tizen.Applications
+    - Synchronous connection request and private file sharing have been added to RPCPort.
+  - Tizen.Network
+    - New Avrcp Control has been added to Bluetooth.
+
+
+### Toolchain
+
+#### New and changed features
+
+- Toolchain configure dumper support has been developed.
+  - "{package}-configure-dump.rpm" in binutils, glibc, and gcc have been provided with “dump_configure“ macro.
+- Build with armv8-a machine option has been added.
+- ASan has been enabled.
+  - Automated build, test with JIRA reporting process have been established.
+  - Libasansi feature for .NET ASan has been back-ported from Tizen 5.5.
+
+#### Fixes
+
+- Patches for CVE-2020-10029 have been back-ported in glibc from upstream.
+
+
+### Machine Learning
+
+#### New and changed features
+
+- ML/Training API (new)
+  - The ability to update and train deep neural networks on Tizen devices has been added.
+  - The ML/Training API allows application developers to describe their own neural network models and train the models or to re-train pre-trained models in run-time.
+- ML/Inference API update
+  - The ability to use various hardware accelerators with Single API has been added.
+  - The ability to fine-tune each element of a pipeline in run-time has been added.
+  - The latency overheads of Single API have been reduced.
+- NNStreamer update
+  - NNStreamer has been upgraded from 1.5.2 to 1.6.0.
+  - AI acceleration hardware support has been added.
+    - ML API may utilize Verisicon-Vivante and Qualcomm-SNPE in addition to previously supported hardware.
+      - You may need to download and install additional adaptors from tizen.org for these additional hardware supports.
+  - The ability to import stream data from or export stream data to Flatbuffers and Protobuffers has been added for distributed or remote pipelines.
+  - The ability to implement and insert tensor-converters for custom data types in run-time has been added.
+- NNTrainer (new)
+  - NNTrainer has been added to allow on-device neural network training, which mainly targets transfer learning mechanisms with a few simple layers.
+  - NNTrainer provides various weight initializers, optimizers, activation functions, and loss functions.
+  - Neural network models may be constructed by applications in run-time or may be loaded from a model file along with pre-trained weight values. The trained weight values can be saved and loaded later.
```

Comment:
```suggestion  
  - Neural Network models can be constructed by applications in run-time or may be loaded from a model file along with pre-trained weight values. Trained weight values can be saved and loaded later.

Reply:
@mijin85cho I didn't change it purposefully as there **may** be some instance when the file is loaded from a model file.
With **can** there is more possibility & with may there is a little lesser possibility. 

We can go with your suggestion, thanks

Reply:
Looks better. Thanks.